tools/plotjuggler/juggle.py

```diff
@@ -13,7 +13,11 @@
 
 juggle_dir = os.path.dirname(os.path.realpath(__file__))
 
-def load_segment(segment_name):
+def load_segment(segment_name, cache):
+  if cache:
+    env = os.environ
+    env["FILEREADER_CACHE"] = "1"
```

Comment:
just set the environment variable instead of adding the --cache flag, like this:
`FILEREADER_CACHE=1 ./juggle.py <route>`